### ISSUE TITLE
Hacktoberfest is going to finish! 

### ISSUE THREAD

**Issue by imaprincess** [imaprincess/Algorithms-4-everyone#86]
Have you completed your 4 PR yet? If not, don't worry! This repo is full of things to do. 
In particular, in this month we collected lot of algorithms - sadly, most of them have implementations in only one or two languages - not good 😔
These are all the algorithms we have in the repo: for this last week, **you can't add new algorithms, you can only write algorithms that are already here in different languages**. Please add them in the correct folder! I will no longer accept PR that aren't following the right method to add the file (look at the other file, you have to insert your code in the correct section and in a folder called as the language you are using), because we spent lot of time cleaning the mess that someone created! (Thanks sooo much @khokharnikunj8 💞) 
So, give a look at the repo and improve it - you can add files in the language you want! 
Do you need some more help? Look at [Dynamic Programming algorithms](https://github.com/imaprincess/Algorithms-4-everyone/tree/master/Algorithms/Dynamic%20Programming) 

Happy Hacktoberfest! 🎃

**Comment by ferrelucas** [comment]
Hi there @imaprincess, I have just run into this repo and I am ready to contribute!

**Comment by sanghaisubham** [comment]
HI @imaprincess ,I am ready to contribute some important Dynamic Programming and Graphs algorithms in Java and C++, as I am able to see ,many algo's are still missing in the repo

**Comment by imaprincess** [comment]
@sanghaisubham you're right, lot of algorithms are missing but for now if you add them I can't assure you that I have time to review them before doing the commit, so: if you want, you can do the pr (it counts for hacktoberfest even if I don't accept them), but probably I will merge them in the long future, I'm sorry :c

**Comment by edujtm** [comment]
Hi @imaprincess, I'd like to help with implementations of shortest path algorithm in Python, LinkedList and radix sort in Cpp. I don't have them implemented right now, but by tomorrow I should have them with some test cases to assure it's working properly.

**Comment by sanghaisubham** [comment]
@imaprincess I will try to add as much Algo's possible from my side .. Whenever u are free please have a look and inform for the necessary updates

**Comment by jsegarra3** [comment]
Hey @imaprincess I'll try to contribute some Ruby versions of the algos

**Comment by abdulsamad234** [comment]
@imaprincess would work on swift versions

**Comment by khokharnikunj8** [comment]
@imaprincess  Thanks for mentioning. I would like to continue contribution.

**Comment by imaprincess** [comment]
@abdulsamad234 sure! c: 
@khokharnikunj8 perfect ahah

**Comment by nicolefffe** [comment]
hi! I'm working on some javascript algorithms :) thanks for the opportunity!

**Comment by SolangeUG** [comment]
Hi @imaprincess !

I'm new to Hacktoberfest and would like to contribute a few algorithms in Java. Thanks for the opportunity :-)

**Comment by Pujolsluis** [comment]
Hello @imaprincess i would like to contribute with the implementation of various algorithm's in C++. Thanks for the opportunity of collaborating :P

**Comment by chaostools** [comment]
added stack implementation in python PR #135

**Comment by jasm123** [comment]
I'm contributing for the first time and would like to contribute some algorithms in Java!

**Comment by SuranjanDaw** [comment]
Hello, @imaprincess I have added Stack and Queue implementation in C. Please review it. #299
